bokeh/models/renderers.py

```diff
@@ -45,6 +48,28 @@ def _check_bad_column_name(self):
         if missing:
             return "%s [renderer: %s]" % (", ".join(sorted(missing)), self)
 
+    @validation.warning(COLON_IN_CATEGORY_LABEL)
+    def _check_colon_in_category_label(self):
+        if not self.glyph: return
+        if not self.data_source: return
+        vm = self.glyph.vm_serialize()
+        labels = (label for label in ['x', 'y']
+                  if label in vm and 'field' in vm[label])
```

Comment:
@bryevdv I think way of filtering out labels doesn't matter.
Should I handle `xs`, `ys` separately, because they are lists of lists?